internal/utils/config.go

```diff
@@ -300,9 +305,16 @@ type (
 	}
 
 	storage struct {
-		Enabled       bool        `toml:"enabled"`
-		Image         string      `toml:"-"`
-		FileSizeLimit sizeInBytes `toml:"file_size_limit"`
+		Enabled       bool                 `toml:"enabled"`
+		Image         string               `toml:"-"`
+		FileSizeLimit sizeInBytes          `toml:"file_size_limit"`
+		S3Credentials storageS3Credentials `toml:"s3_credentials"`
```

Comment:
```suggestion
		S3Credentials storageS3Credentials `toml:"-"`
```

We probably don't want users to override these in local dev.